### DISCUSSION TITLE
Accordion tabs not closing

### DISCUSSION THREAD

**Discussion by fredbouin** [alpinejs/alpine#3804] [7. Bugs (archived)]
Something changed with my accordion component in the latest release of alpine 3.13.1. When going from one accordion tab to the next the previous one doesn't close anymore.

I have an accordion component based off the one in the tutorials from the alpine UI components. You can see the bug in the page for the component itself https://alpinejs.dev/component/accordion 

If you click on "question 2", "question 1" should close, and vice versa. Not sure where the error is, will stick with alpine 3.13 for now as it works correctly on this version.

**Comment by ekwoka** [comment]
Okay this is a regression in code I refactored.

`this` doesn't refer to the full component stack in setters, only the most local context.

In the old code it didn't always allow that and making it always allow it failed tests and not allowing it didn't fail anything.

I'll figure it out :)

**Comment by ekwoka** [comment]
#3807
